templates/glauth.toml.j2

```diff
@@ -0,0 +1,28 @@
+#################
+# General configuration.
+debug = true
+
+#################
+# Server configuration.
+[ldap]
+enabled = true
+listen = "0.0.0.0:{{ ldap_port }}"
+
+[behaviors]
+IgnoreCapabilities = false
+LimitFailedBinds = true
+NumberOfFailedBinds = 3
+PeriodOfFailedBinds = 10
+BlockFailedBindsFor = 60
```

Comment:
Not necessary for this, but likely these would be ideal candidates for config values. I can see some potential config-charms being used to provide configuration for backends and the charm itself providing configurations for these various scenarios (I believe these are global and not backend dependent).

Reply:
Sounds like we will be covering this in a glauth config spec to come.